main.go

```diff
@@ -95,9 +95,24 @@ func main() {
 		},
 		cli.StringFlag{
 			Name:   "commit.author",
-			Usage:  "git author name",
+			Usage:  "git author",
 			EnvVar: "DRONE_COMMIT_AUTHOR",
 		},
+		cli.StringFlag{
```

Comment:
You should take https://github.com/drone/drone-plugin-starter/blob/master/main.go#L102-L116 as an example how the flags should look like.